Base/res/ladybird/default-config/BrowserContentFilters.txt

```diff
@@ -121,37 +135,72 @@ adpushup.com
 adrecover.com
 adrizer.com
 adroll.com
+ads-api.tiktok.com
+ads-api.twitter.com
+ads-dev.pinterest.com
+ads-sg.tiktok.com
 ads-twitter.com
 ads.bidstreamserver.com
+ads.facebook.com
+ads.imeem.com
 ads.linkedin.com
 ads.mobillegends.net
 ads.nexage
+ads.pinterest.com
 ads.playground.xyz
+ads.pointdrive.linkedin.com
+ads.reddit.com
 ads.resetsrv.com
 ads.servebom.com
+ads.tiktok.com
+ads.trafficjunky.net
 ads.yahoo.com
+ads.yap.yahoo.com
+ads.youtube.com
+ads2.contentabc.com
+ads30.adcolony.com
 adsafeprotected.com
 adscale.de
+adsclk.com
+adsco.re
+adserver.adtech.advertising.com
+adserver.socialmint.news
+adserver.unityads.unity3d.com
+adservetx.media.net
 adservice.google.
+adservice.google.co.in
+adservice.google.com
```

Comment:
Looking at `Web::ContentFilter::is_filtered()`, these two should already be included by the line before it?